apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -943,6 +940,24 @@ suite('ActivityHandler', function() {
         );
       }).then(done,done);
     });
+
+    test('existing thread with specified body, navigate to composer',
```

Comment:
__question:__ what is the value in this test if we already have very similar one ```'Should move to the composer and set activity``` (with added ```sinon.assert.notCalled(MessageManager.findThreadFromNumber)``` in the test above it should not matter whether thread exists or not)?